simpleFunctions.js

```diff
@@ -0,0 +1,230 @@
+'use strict'
+
+const ratFract = require('./rationalFractions.js')
```

Comment:
This file looks unrelated to the other ones? (it just defines some matrix manipulation functions not used anywhere else)